samples/notebooks/prediction/SDK_Custom_Predict.ipynb

```diff
@@ -1,1263 +1,1381 @@
+    "id": "8phJWBhe7u9u",
+    "tags": []
+   },
+   "source": [
+    "#### Set up credentials\n",
+    "\n",
+    "Setting up credentials is only required to run the custom serving container locally. Credentials set up is required to execute the `Predictor`'s `load` function, which downloads the model artifacts from Google Cloud Storage.\n",
+    "\n",
+    "To access to the Google Cloud Storage in your project, you can use one of the following:\n",
+    "\n",
+    "1. A user account, which is associated with a Google account and is intended to represent a specific individual on your project.\n",
+    "\n",
+    "1. A service account, which is intended to represent an application or a process instead of a person. Use service accounts in your automated build, test, and deployment processes, especially if multiple developers can run these processes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Option 1. Use Google user credentials**\n",
+    "\n",
+    "First authorize gcloud to access the Cloud Platform with Google user credentials. This step requires to put user inputs. If you are using a non-interactive shells, you should open a terminal to run this command. If you are using Vertex Workbench, you will need to open [a terminal](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html) to run this command.\n",
+    "\n",
+    "This command will print the credential file as `Credentials saved to file: [CREDENTIALS_FILE]`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud auth application-default login"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Specify the credential file, which is a json file, as the path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CREDENTIALS_FILE = \"[CREDENTIALS_FILE]\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Grant roles to the user account. Since we will be using this user account to download the model artifacts from Google Cloud Storage, we grant the user account **Storage Object Admin** role. See [IAM roles for Cloud Storage](https://cloud.google.com/storage/docs/access-control/iam-roles) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "USER_ACCOUNT = \"[USER_ACCOUNT]\"  # @param {type:\"string\"}\n",
+    "\n",
+    "!gcloud projects add-iam-policy-binding $PROJECT_ID \\\n",
+    "    --member=user:$USER_ACCOUNT --role=roles/storage.objectAdmin"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Option 2. Use Google Service Account credentials**\n",
+    "\n",
+    "First enable the IAM API if it's not already enabled."
```

Comment:
Let's add something here that says...

Skip this if you've already completed Option 1 above.

Reply:
Done.